docs/logging-install.md

```diff
@@ -0,0 +1,148 @@
+**This document discusses setting up of logging infrastructure in the OpenShift
```

Comment:
I think we should mention elasticsearch when we mention `logging infrastructure`, WDYT?